readme.md

```diff
@@ -43,10 +43,10 @@ del.sync(['public/assets/**', '!public/assets', '!public/assets/goat.png']);
 
 ## API
 
-### del(patterns, [options], callback)
+### del(patterns, [options])
 ### del.sync(patterns, [options])
 
-The async method gets an array of deleted paths as the second argument in the callback, while the sync method returns the array.
+The async method gets Promise yielding an array of deleted paths, while the sync method returns the array.
```

Comment:
Yeah, resolves is more consistent with the spec and most docs.

On Sun, Aug 30, 2015 at 10:57 AM Sindre Sorhus <notifications@github.com>
wrote:

> In readme.md
> <https://github.com/sindresorhus/del/pull/29#discussion_r38274084>:
>
> >  ### del.sync(patterns, [options])
> >
> > -The async method gets an array of deleted paths as the second argument in the callback, while the sync method returns the array.
> > +The async method gets Promise yielding an array of deleted paths, while the sync method returns the array.
>
> For consistency with the globby readme, can you use resolves instead of
> yielding? (Or is yielding more correct?).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sindresorhus/del/pull/29/files#r38274084>.
>